### ISSUE TITLE
Always display winrate for black?

### ISSUE THREAD

**Issue by nicolasrochette** [featurecat/lizzie#248]
This is a trivial one. Would it be possible to always display Black's estimated winrate (on the board, the winrate graph is already always from Black's perspective), instead of alternating between those of the two players? The current setting makes it really hard to follow who's gaining something for a series of moves. Yes, I have no problem doing the math but it's still a pain, so most of the time I just end up ignoring the winrate during White's turn, despite the fact that I then only get information once every two moves.

I don't know the code but possibly this could be changed trivially. And I would like to argue that the change could be just made without any additional developments for now, and that in the future, conditioned on people's reactions, a toogle option could be added.

(edit: Also, thank you for all the work. It's amazingly useful)

**Comment by featurecat** [comment]
isn't that already a lizzie.properties option?

**Comment by featurecat** [comment]
if I recall correctly. dfannius added that feature?

**Comment by nicolasrochette** [comment]
That is correct, sorry, I didn't know this config file existed. I changed the option (`"win-rate-always-black": true`, in `ui`) and it behaves as expected.
